### DISCUSSION TITLE
Having Problems with Nemesis Engine

### DISCUSSION THREAD

**Discussion by AimlessRain** [community/community#37842] [Other Feature Feedback, Questions, & Ideas]
I'm pretty new to Mod Organizer 2 and I tried to get Nemesis to update, but I run into Error 6006 and the update won't make it past 3-4% without crashing. Super annoying. I reinstalled the mod in various ways and changed the directory to no avail. Need help, thanks.

**Comment by jfullstackdev** [comment]
they have their community 

https://forums.nexusmods.com/
